dali/test/python/test_operator_readers_nemo_asr.py

```diff
@@ -257,3 +257,25 @@ def _testimpl_nemo_asr_reader_pad_last_batch(batch_size):
 
   # Trying to catch race conditions (A lot of samples in the batch to be replicated)
   yield _testimpl_nemo_asr_reader_pad_last_batch, 128
+
+def test_read_idxs(batch_size=10):
+  @pipeline_def(device_id=0, num_threads=4)
+  def nemo_asr_reader_read_idxs(reader_seed=12345):
+    audio, idx = fn.readers.nemo_asr(manifest_filepaths=[nemo_asr_manifest], random_shuffle=True, seed=reader_seed,
+                                     read_sample_rate=False, read_text=False, read_idxs=True)
+    return audio, idx
+  pipe1 = nemo_asr_reader_read_idxs(batch_size=batch_size)
+  pipe1.build()
+  pipe2 = nemo_asr_reader_read_idxs(batch_size=batch_size)
+  pipe2.build()
```

Comment:
What is the value in comparing two, identical piepelines?

Reply:
I wanted to check that the order of indices is the same for the same seed

Reply:
Ok

Reply:
So how about setting the same see in the L267 and L269 explicitly?

Reply:
done